parsers/jscript.c

```diff
@@ -1941,6 +1952,17 @@ static void initialize (const langType language)
 {
 	Assert (ARRAY_SIZE (JsKinds) == JSTAG_COUNT);
 	Lang_js = language;
+
+	TokenPool = objPoolNew (16, (objPoolCreateFunc)newPoolToken,
+		(objPoolDeleteFunc)deletePoolToken, (objPoolClearFunc)clearPoolToken);
```

Comment:
Now the functions are specific to the pool, they could have the appropriate signature, and the conversion be done internally.  Doesn't matter too much, but could allow the compiler to check those in case they ever change.